cross_compile/Dockerfile_ros

```diff
@@ -72,35 +81,48 @@ RUN python3 -m pip install -U \
     pytest-cov \
     pytest-runner \
     setuptools \
-    vcstool
+    vcstool; \
+  fi
 
-# Install Fast-RTPS dependencies
-RUN apt-get update && apt-get install --no-install-recommends -y \
+# Install Fast-RTPS dependencies for ROS 2
+RUN if [[ "${ROS_VERSION}" == "ROS2" ]]; then \
+    apt-get update && apt-get install --no-install-recommends -y \
         libasio-dev \
         libtinyxml2-dev \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/*; \
+  fi
 
 # Run arbitrary user setup
 COPY user-custom-setup .
 RUN chmod +x ./user-custom-setup && \
     ./user-custom-setup && \
     rm -rf /var/lib/apt/lists/*
 
-# Setup ROS2 workspace
-COPY ${ROS2_WORKSPACE}/src /ros2_ws/src
-WORKDIR /ros2_ws
+# Setup ROS workspace
+COPY ${ROS_WORKSPACE}/src /ros_ws/src
+WORKDIR /ros_ws
 
-# Run rosdep to get dependencies for the copied-in ROS2 workspace
+# Run rosdep to get dependencies for the copied-in ROS workspace
 ENV ROSDEP_SKIP_KEYS="console_bridge fastcdr fastrtps libopensplice67 libopensplice69 rti-connext-dds-5.3.1 urdfdom_headers"
-RUN rm -f /etc/ros/rosdep/sources.list.d/20-default.list    # In case of cached image.
+
+RUN rm -f /etc/ros/rosdep/sources.list.d/20-default.list
 RUN rosdep init
-RUN rosdep update && \
+RUN rosdep update
+
+RUN if [[ "${ROS_VERSION}" == "ROS2" ]]; then \
     apt-get update && \
-    rosdep install --from-paths src \
-        --ignore-src \
-        --rosdistro ${ROS_DISTRO} -y \
-        --skip-keys "${ROSDEP_SKIP_KEYS}" \
-    && rm -rf /var/lib/apt/lists/*
+        rosdep install --from-paths src \
+            --ignore-src \
+            --rosdistro ${ROS_DISTRO} -y \
+            --skip-keys "${ROSDEP_SKIP_KEYS}" \
+        && rm -rf /var/lib/apt/lists/*; \
+  else \
+    apt-get update && \
```

Comment:
the only change here is the value of skip_keys - can we just leave it as a non-conditional?

Reply:
Tested with `kinetic` and `melodic`, works w/o the conditionial.